modules/channel-telegram/src/senders/common.ts

```diff
@@ -0,0 +1,39 @@
+import { ChannelSender } from 'common/channel'
+import { TelegramContext } from '../backend/typings'
+
+export class TelegramCommonSender implements ChannelSender<TelegramContext> {
+  get channel(): string {
+    return 'telegram'
+  }
+
+  get priority(): number {
+    return 0
+  }
+
+  get id() {
+    return TelegramCommonSender.name
+  }
+
+  handles(context: TelegramContext): boolean {
+    return context.handlers?.length > 0
```

Comment:
Not sure why you use the '?' operator here since handlers is not an optional field?

Reply:
Good point